tools/vf-frctl-theme/views/layouts/skeleton.njk

```diff
@@ -19,7 +19,7 @@
   </script>
   {% include 'partials/stylesheets.njk' %}
 </head>
-<body>
+<body class="vf-body">
```

Comment:
Moving `vf-body` up to the body element "traps" some of the grid and container layouts; example: /components/detail/vf-grid-page

Maybe conditionally using `vf-u-grid--reset` somewhere is a way around it.

Reply:
what would you say is in correct about `/components/detail/vf-activity-group` I'm guessing the lack of full-width colour block.

I think this goes back to an old idea that started to get implemented with `vf-inlay`s `vf-masthead` so that we use `:before` and `:after` to do the full width bits. 

Also, I think the grids aren't _really_ components but 'styles' as several other design systems categorise them (along with some our simple element components). With Design Tokens as their own thing.

I'm writing a little proposal about this - so I'll work on getting that to all

Reply:
Just a note that the other wide components also get impacted, like vf-header vf-masthead and vf-footer

Reply:
Yep :)